src/main/java/com/dev/cinema/model/MovieSession.java

```diff
@@ -18,9 +18,9 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.EAGER)
     private Movie movie;
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
I could rewrite the _**shoppingCartService.getAll()**_ method using two subqueries to get all of the required data (for example, for the _**toString()**_ method), (_shopping cart_ has a _ticket_ that has a _movie_ _session_ that has a _movie_ etc), but in the context of the business logic of this project I decided to choose the **_fetch type EAGER_** because I'm planning to use all of the information in the web part of the project.

Reply:
так для ManyToOne дефолтный FetchType и так, если я не ошибаюсь, EAGER. или нет?

Reply:
Да, потому я и удалила)

Reply:
Я так понимаю, Миша имел в виду, что в аннотации можно ничего не писать в скобках

Reply:
Ну вот я это и удалила)))